services/QuillLMS/app/assets/stylesheets/pages/my_lessons.scss

```diff
@@ -104,9 +104,6 @@
         }
       }
       .data-table-body .data-table-row .data-table-row-section {
-        .customize-link, .download-link {
-          margin-left: 22px;
-        }
```

Comment:
The reason I added this margin was so that the focus indicators for these buttons didn't get cut off on the left side, so we should keep these

Reply:
They don't any longer -- that's part of what I was trying to correct here. By using padding instead of margin for the spacing of these columns, we eliminate the funkiness of where the border actually lies.
<img width="1233" alt="Screenshot 2024-12-17 at 10 33 39 AM" src="https://github.com/user-attachments/assets/09733cf3-097c-469f-a09e-13556a5a6356" />

Reply:
ohhh-- awesome! thanks for fixing that